virt_lightning/shell.py

```diff
@@ -230,6 +229,35 @@ def list_from_yaml_file(value):
     )
     fetch_parser.add_argument("distro", help="Name of the VM image", type=str)
 
+    # custom configuration parser
```

Comment:
I'm not sure we need this to resolve the problem if we document how to edit the configuration. This being said the idea I like `--output` parameter, can we take the time to speak about first?